### ISSUE TITLE
Pre-trained weights

### ISSUE THREAD

**Issue by ethancohen123** [microsoft/batch-effects-normalization#7]
Hi,
Is it possible to have access to the pre-trained weights of some models ?

**Comment by alexxijielu** [comment]
Sorry for the delay on this. We've uploaded some of the pretrained weights here:
https://drive.google.com/file/d/1ISgd1EDegiyy7TSyzwGhfVTQCzMNXSrC/view

- image_supervised.pt corresponds to the best model in Table 1 of our paper.
- cell_minclr_k5.pt, cell_simclr_k5.pt, and cell_supervised.pt correspond to the best models in each row of Table 2 of our paper

**Comment by ethancohen123** [comment]
Thanks !
